website/templates/project/view_file.mako

```diff
@@ -214,7 +215,8 @@
       window.contextVars.file.urls.external = window.contextVars.file.extra.webView;
     </script>
 
-    <link href="/static/css/pages/file-view-page.css" rel="stylesheet">
+    <link href="/static/css/pages/file-view-page.css" rel="stylesheet" />
+    <link href="/static/css/pages/share-buttons.css" rel="stylesheet" />
```

Comment:
Since these styles are tightly tied to the mithril components you created, it would make sense to import this css file (using `require`) in `socialshare.js`.